metadata.json

```diff
@@ -18,7 +18,7 @@
     },
     {
       "name": "puppetlabs/apache",
-      "version_requirement": ">= 1.6.0 < 7.0.0"
+      "version_requirement": ">= 8.0.0 < 9.0.0"
```

Comment:
It wasn't needed to change the lower bound so strict. This can cause problems for users who haven't updated all their yet. The old versions accepted integers just fine.

Reply:
I must agree that the lower bound is quite aggressive 😄